README.md

```diff
@@ -199,6 +199,16 @@ application.
 
 A simple but secure example is detailed in `README.google_oauth2.md`.
 
+### Routemaster client
+
+[routemaster-client](https://github.com/deliveroo/routemaster-client) comes as a dependency with a base events publisher.
+
+To publish events onto the bus from your app, you must:
+
+* Make sure you have the `ROUTEMASTER_ENABLED` env var set to `true`.
+* Add the `publish_lifecycle_events` class method to the relevant model(s).
+* Create publisher file(s) with `url` method and others if applicable.
```

Comment:
Maybe a detailed example would be useful, as we did in `README.google_oauth2.md`?